### ISSUE TITLE
Linux Mint's Sticky app doesn't re-open after not showing the notes

### ISSUE THREAD

**Issue by 4r7if3x** [NixOS/nixpkgs#352696]
### Describe the bug

The app opened once without showing any of the notes. Once I closed the app, I could not open it again, no errors shown.

### Steps To Reproduce

1. Install the `sticky` package via Home-Manager:

    ```nix
    { ... }:

    {
      home.packages = with pkgs; [
        sticky
      ];
    }
    ```

2. Try to open the app.
3. If opened, see if notes are visible.

### Expected behavior

The app should open when its icon in the launcher is clicked on, or when `sticky` command is ran into the Shell. Once opened, the notes should be visible both detached and inside the main window.

### Notify maintainers

@linsui @bobby285271 @collinss @clefebvre

### Metadata

```bash
[user@host:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.58, NixOS, 24.11 (Vicuna), 24.11.20241023.2768c7d`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.9`
 - channels(root): `"nixos-24.05"`
 - nixpkgs: `/nix/store/m8u4z9t2q1b3e5j6f0r7w8y2x9c4v6b1-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by bobby285271** [comment]
> no errors shown

I don't think it will print anything if it is already running (e.g. check `pgrep -fa sticky`)? To actually get some logs, you will likely need to kill the existing one first I think. And to open the already running sticky I think you will have to open it from the tray.

<img src="https://github.com/user-attachments/assets/5e368d3a-21a8-4400-9c5d-f4b2ae8fb196" width=30%>

**Comment by 4r7if3x** [comment]
@bobby285271 I previously did kill its process before re-launching it. I also never had its icon in the top bar although legacy app indicators are enabled.

Anyways, I've switched to the `sticky-notes` package instead...
